### PR TITLE
Add News.md for OME Model 5.6.1 release

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/X7mjq1vn/209-ome-model-56

### CODE REVIEW

**Comment by hflynn** [comment]
See https://github.com/hflynn/ome-model/blob/d25ec1984f31565564757bfc0e8d4773e8b07632/NEWS.md for rendered content for review

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+5.6.1 (2019-02-17)
```
Comment (mtbc) [inline NEWS.md]: Really 2019?

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+* OME-XML C++: insert missing links [#30](https://github.com/ome/ome-model/pull/30)
+
+5.5.0 (2017-02-10)
+------------------
+
+* Enable C++ 11 support [#10](https://github.com/ome/ome-model/pull/10)
+* Add minimal C++ 11 features [#25](https://github.com/ome/ome-model/pull/25)
+* OME-XML performance improvements
+  [#28](https://github.com/ome/ome-model/pull/28)
+* Cmake: obtain release version from Maven
```
Comment (mtbc) [inline NEWS.md]: CMake

**Comment by rleigh-codelibre** [comment]
Looks fine to me.  As I've noted for other NEWS.md files before, I do dislike putting the release date alongside the version in the heading.  It's of lesser importance and could be put on the line below the heading.

**Comment by hflynn** [comment]
Current formatting matches the ome-common NEWS.md (and the layout in the docs version histories) - I think it's unnecessary effort to reformat this and that one so they match going forward so I suggest we leave it as-is. If it really bothers you @rleigh-codelibre please feel free to open a PR against this and on ome-common to suggest your preferred formatting.

**Comment by sbesson** [comment]
Thanks, merging and releasing as 5.6.1 then.

**Pull request merged**